src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar
+
+import camp.nextstep.edu.missionutils.Randoms
+
+class Car(private var name: String) {
+    private var distance = 0
+    private var cluster:Cluster = Cluster()
+
+    fun getName():String{
+        return name
+    }
+
+    fun getDistance():Int{
```

Comment:
함수의 리턴타입을 선언할 때 띄어쓰기도 컨벤션에 포함됩니다.

`ctrl` + `alt` + `l` 을 하면 이런 띄어쓰기나 줄넘김이 자동으로 되니까 한번 사용해보세요!

```suggestion
    fun getDistance(): Int{
```

Reply:
지적 감사합니다ㅠㅠ IDE 코드 컨벤션 도구설정을 다시 확인해봐야겠어요 ㅠ